[2022-11-02 14:38:11,910] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: xetra_DAG.run_xetra_etl manual__2022-11-02T14:35:52.496941+00:00 [queued]>
[2022-11-02 14:38:11,940] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: xetra_DAG.run_xetra_etl manual__2022-11-02T14:35:52.496941+00:00 [queued]>
[2022-11-02 14:38:11,940] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-11-02 14:38:11,941] {taskinstance.py:1357} INFO - Starting attempt 2 of 2
[2022-11-02 14:38:11,950] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-11-02 14:38:12,454] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): run_xetra_etl> on 2022-11-02 14:35:52.496941+00:00
[2022-11-02 14:38:12,466] {standard_task_runner.py:52} INFO - Started process 179 to run task
[2022-11-02 14:38:12,483] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'xetra_DAG', 'run_xetra_etl', 'manual__2022-11-02T14:35:52.496941+00:00', '--job-id', '139', '--raw', '--subdir', 'DAGS_FOLDER/xetra_DAG.py', '--cfg-path', '/tmp/tmpt2axto3_', '--error-file', '/tmp/tmpb3djzj5i']
[2022-11-02 14:38:12,486] {standard_task_runner.py:80} INFO - Job 139: Subtask run_xetra_etl
[2022-11-02 14:38:13,333] {task_command.py:369} INFO - Running <TaskInstance: xetra_DAG.run_xetra_etl manual__2022-11-02T14:35:52.496941+00:00 [running]> on host 0163e0790208
[2022-11-02 14:38:14,180] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=xetra_DAG
AIRFLOW_CTX_TASK_ID=run_xetra_etl
AIRFLOW_CTX_EXECUTION_DATE=2022-11-02T14:35:52.496941+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-02T14:35:52.496941+00:00
[2022-11-02 14:38:14,182] {logging_mixin.py:115} INFO - /opt/***/meta_file.csv
[2022-11-02 14:38:47,009] {python.py:173} INFO - Done. Returned value was: None
[2022-11-02 14:38:47,146] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=xetra_DAG, task_id=run_xetra_etl, execution_date=20221102T143552, start_date=20221102T143811, end_date=20221102T143847
[2022-11-02 14:38:47,388] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-11-02 14:38:48,958] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
